src/main/java/tony/Ui.java

```diff
@@ -0,0 +1,35 @@
+package tony;
+
+/**
+ * The Ui class handles user interface-related operations.
+ */
+public class Ui {
+
+    /**
+     * Displays a greeting message when the program starts.
+     */
+    public static void greeting() {
+        String greeting = "_______________________\n"
+                + "what is up dawg! I'm tony.Tony!\n"
+                + "What can I do for you mate?\n"
+                + "_________________________\n";
+        System.out.println(greeting);
+    }
```

Comment:
Should line() method be used for consistency?
```suggestion
        String greeting = "What is up dawg! I'm tony. Tony!\n"
                + "What can I do for you mate?\n";
        line();
        System.out.println(greeting);
        line();
    }
```